service_container/service_closures.rst

```diff
@@ -58,6 +58,9 @@ argument of type ``service_closure``:
             App\Service\MyService:
                 arguments: [!service_closure '@mailer']
 
+                # In case the dependency is optional
+                # arguments: [!service_closure '@?mailer']
```

Comment:
I'm not sure if this works actually

Reply:
I checked and it's ok, if `mailer` service doesn't given null is pass instead